### PR TITLE
Update repository

### PR DESCRIPTION
**Pull request by jlmakes**
Spotted a few code blocks without syntax highlighting, and touched things up a bit.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,28 @@
-MIT License
+BSD 3-Clause License
```
Comment (kriszyp) [inline LICENSE]: Is this a better license to use for Alkali?

Comment (jlmakes) [reply LICENSE]: I'm not sure about that, but I was trying to follow more closely what it's in `package.json`

**Comment by jlmakes** [comment]
Since the `LICENSE` file was correct after all, I force pushed an update to `package.json` to update the license field.

**Comment by kriszyp** [comment]
Thank you!

**Pull request merged**